### PR TITLE
Update CCI to be string array

### PR DESCRIPTION
**Pull request by jasonodonnell**
Issue here: #6 

One of the controls has multiple CCIs, which requires a string array.  The rest of the controls are just normal strings.  This makes an inconsistency with the json output and makes it very hard to parse the output in statically typed languages.

This patch changes all CCI to be a string array to avoid inconsistency.

### CODE REVIEW

**Comment by steve-hetzel** [comment]
This all makes sense to me - very straight-forward PR.  Thanks, @jasonodonnell .

**Review by sharmay — APPROVED** [review]
